e2e/package.json

```diff
@@ -18,6 +20,7 @@
     "mocha": "^6.1.4",
     "selenium-webdriver": "^3.6.0",
     "ts-node": "^8.0.3",
+    "tslint": "^5.16.0",
```

Comment:
I have found that we have CQ for "5.10.0" tslint version
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16944

Reply:
Version changed to ```5.10.0```.